addons/html_editor/static/src/plugin.js

```diff
@@ -38,7 +38,9 @@ export class Plugin {
     setup() {}
 
     isValidTargetForDomListener(ev) {
-        return !isProtecting(ev.target) && (!isProtected(ev.target) || isUnprotecting(ev.target));
+        return (
+            !isProtecting(ev.target) && (!isProtected(ev.target) || isUnprotecting(ev.target))
+        ) || this.getResource("editable_node_predicates").some(predicate => predicate(ev.target));
```

Comment:
You're adding a way to bypass all the "protecting"/"protected" checks... :/

Reply:
Oops :grimacing: